recipes/interpolation/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: interpolation
+  version: "0.1.8"
+source:
+  git_url: https://github.com/econforge/interpolation.py
+  git_tag: v0.1.8
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  preserve_egg_dir: True
+  number: 1
+  skip: True  # [py>35]
```

Comment:
python 3.6 gave me an error saying that numpy1.9 and python3.6 were incompatible